### PR TITLE
Drop `follow-file-renames` feature

### PR DESCRIPTION
**Pull request by FloEdelmann**
It was implemented by GitHub, see https://github.blog/changelog/2022-06-06-view-commit-history-across-file-renames-and-moves/

## Test URLs

* [Renamed file on main branch](https://github.com/MicrosoftDocs/azure-docs/commits/main/articles/app-service/tutorial-connect-msi-sql-database.md)
* [Renamed file on branch with slashes](https://togithub.com/refined-github/sandbox/commits/branch/with/slashes/renamed.md)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -114,9 +114,6 @@ GitHub Enterprise is also supported: [How to enable it](https://fregante.github.
 		<th width="50%">
 			<p><a title="conversation-activity-filter"></a> Lets you hide every event except comments or unresolved comments in issues and PRs
 			<p><img src="https://user-images.githubusercontent.com/1402241/109592127-5f922200-7ad4-11eb-8dfa-1d80fb28e70e.png">
-		<th width="50%">
-			<p><a title="follow-file-renames"></a> Enhances files’ commit lists navigation to follow file renames
-			<p><img src="https://user-images.githubusercontent.com/1402241/54799957-7306a280-4c9a-11e9-86de-b9764ed93397.png">
```
Comment (FloEdelmann) [inline readme.md:119]: Should another feature be highlighted here now?

Comment (fregante) [reply readme.md:119]: Let's add `select-notifications` as part of this PR

**Comment by FloEdelmann** [comment]
I couldn't find a case where GitHub's feature doesn't work and RGH's does.

File: readme.md (hunk 1 above)
Comment (FloEdelmann) [reply readme.md]: Done in [`917e4eb`](https://github.com/refined-github/refined-github/pull/5686/commits/917e4eb83a6fef68f9e9de916c85e0ce6ceccdbd).

**Comment by fregante** [comment]
@FloEdelmann Thanks!

I just noticed that `quick-mention` isn't appearing on most of your comments 😰 

Screen Shot

**Comment by kidonng** [comment]
> I just noticed that `quick-mention` isn't appearing on most of your comments 😰

That's working as expected, reviews and minimized comments are excluded, though I'm not sure about the top comment.

**Comment by fregante** [comment]
This is not working as expected:

Screen Shot

It should appear in more places:

- reviews
- first comment

But yeah one of those is expected:

- https://github.com/refined-github/refined-github/pull/4609

**Comment by simonkotwicz** [comment]
Is it possible to add this back for GitHub enterprise users who don't have this feature yet?

**Comment by fregante** [comment]
This feature was added in GitHub Enterprise 3.7.0, which was released 4 months ago: https://docs.github.com/en/enterprise-server@3.7/admin/release-notes#3.7.0-features

**Comment by simonkotwicz** [comment]
I don't have control over the version of GitHub enterprise I am using unfortunately. Depending on the size and flavor of Github enterprise, updates can require significant downtime. And I believe the GitHub team  strongly recommends only upgrading one minor version at a time which slows things down as well.

**Pull request merged**